src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -361,19 +361,13 @@ const MarkerHeatmap = ({ experimentId }) => {
         </Tabs>
       </Panel>
       <Panel header='Select data' key='select-data'>
-        <Space direction='vertical' size='small'>
-          <p>Select cell sets to show in the heatmap:</p>
-          <Select
-            value={{
-              value: config.selectedCellSet,
-            }}
-            onChange={changeClusters}
```

Comment:
is the variable `changeClusters` needed anymore? Did you check in the code and delete its declaration if it is no longer used?